### ISSUE TITLE
New alerts should be displayed with significance on the alerts screen

### ISSUE THREAD

**Issue by walkeriniraq** [walkeriniraq/twitarr#203]

**Comment by tasermonkey** [comment]
All of the alerts includes a timestamp, could we make some helper function in the javascripts to test 'timestamp' that can be used to detect 'new' -vs- 'old' ?

**Comment by walkeriniraq** [comment]
Yeah - if you pass the last check timestamp, I can add a class to those elements.
